mongodump/oplog_dump.go

```diff
@@ -158,7 +158,7 @@ func oplogDocumentValidator(in []byte) error {
 	}
 
 	if ok && nsStr == "admin.system.version" {
-		return fmt.Errorf("cannot dump with oplog if admin.system.version is modified")
+		return fmt.Errorf("cannot dump with oplog if admin.system.version is modified by %v", raw)
```

Comment:
Doesn't printing a `bson.Raw` just print a `[]byte` slice in hex? I think that might be confusing for any users who see this error.

Reply:
It comes out as text, somehow.  I've looked at the logs. bson.Raw has a String() method so that will get it formatted.

"5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any)."

Reply:
Ah, interesting. I think in that case you should use `%s` instead.